android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -1,3 +1,10 @@
+let _debug = 0;
```

Comment:
```suggestion
var _debug = 0;
```

Unfortunately, the `let` syntax isn't supported by Chrome on our older Android 5.0 devices (SDK 21).
Also for l. 227